openapi/crm/crm.json

```diff
@@ -0,0 +1,381 @@
+          },
+          "name": "resourceTypeCode",
+          "in": "path",
+          "required": true,
+          "description": "The resourceTypeCode, here Contact"
+        }
+      ],
+      "get": {
+        "summary": "Get field schema",
+        "tags": [],
+        "responses": {
+          "200": {
+            "description": "This API follows [Open API schema](https://spec.openapis.org/oas/v3.1.0#schema-object) of structure for response.",
+            "content": {
+              "application/json": {
+                "examples": {
+                  "Get crm field": {
+                    "value": {
+                      "fields": [
+                        {
+                          "externalId": "externalId_xxxx",
+                          "type": "FIELD_TYPE_xxxx",
+                          "title": "Last activity date",
+                          "description": "Date and time of last activity"
+                        },
+                        {
+                          "externalId": "externalId_xxxx_xxxx",
+                          "type": "FIELD_TYPE_xxxxx",
+                          "title": "Mobile phone number",
+                          "description": "Mobile phone number of the contact."
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "User Not Found"
+          }
+        },
+        "operationId": "ListFieldSchema",
+        "description": "Retrieve field schema based on namespace."
+      }
+    },
+    "/{namespace}/{resourceTypeCode}/meta/fields/{fieldId}/options": {
+      "parameters": [
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "namespace",
+          "in": "path",
+          "required": true,
+          "description": "The namespace, PID/AGID"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "resourceTypeCode",
+          "in": "path",
+          "required": true,
+          "description": "The resourceTypeCode, here Contact"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "fieldId",
+          "in": "path",
+          "required": true,
+          "description": "Unique identifier for field"
+        }
+      ],
+      "get": {
+        "summary": "Get field options",
+        "tags": [],
+        "responses": {
+          "200": {
+            "description": "This API follows [Open API schema](https://spec.openapis.org/oas/v3.1.0#schema-object) of structure for response.",
+            "content": {
+              "application/json": {
+                "examples": {
+                  "Get field options": {
+                    "value": {
+                      "options": [
+                        {
+                          "value": "Option 1",
```

Comment:
As I understand the return type in response will have a json like { "options" : [string1, string2...]}

Reply:
We can explore this possibility later.